### PR TITLE
update for manual construction

### PR DESCRIPTION
**Pull request by vidhyamanisankar**
https://jira.ons.gov.uk/browse/SPP-10067

### CODE REVIEW

**Review by BenLatham — APPROVED** [review]

**Review by jchapman68 — APPROVED** [review]

**Review by kingmushroom — APPROVED** [review]

File: editing_and_imputation/imputation/general/technical_specification.md [hunk 1]
```diff
@@ -222,7 +234,8 @@ to the construction link
 
 In link calculations, only responders present in the same group in both
 target and predictive periods shall be used to calculate the ratios. These
-pairs of responders are referred to as matched pairs.
+pairs of responders are referred to as matched pairs. Non responders
+with a manual contruction value are excluded from link calculaion.
```
Comment (gibbardsteve) [inline editing_and_imputation/imputation/general/technical_specification.md:238]: Minor typo - calculation

**Review by gibbardsteve — APPROVED** [review]
Looks good - minor typo

File: editing_and_imputation/imputation/general/technical_specification.md (hunk 1 above)
Comment (vidhyamanisankar) [reply editing_and_imputation/imputation/general/technical_specification.md]: Removed this sentence as previous statement `**only responders** present in the same group in both
target` will cover manual construction contributor as well.

**Review by gibbardsteve — APPROVED** [review]
Approved

**Review by kingmushroom — APPROVED** [review]

**Pull request merged**